src/dmd/backend/rtlsym.d

```diff
@@ -173,6 +173,7 @@ enum
 extern (C++):
 
 nothrow:
+@safe:
```

Comment:
This should be `@trusted`

Reply:
why? Besides, the d style programs rejects all `@trusted:` declarations.

Reply:
Because those are function definition, not declaration, so `@safe` is not checked.
re:besides, we're talking about two functions.

Reply:
Isn't the point of this change to falsely consider everything as @safe until we can actually make them @safe? If so, does it really matter if put trusted or safe?

Reply:
Well those won't show up if you grep for trusted. But since the mangling will not match, I suppose we can just ignore it for now...